scripts/standardising.py

```diff
@@ -50,11 +51,11 @@ def run_standardising(
 
     Returns:
         a list of FileTiff wrapper
+        :param gdal_version:
```

Comment:
Should be in the Args list?

Reply:
Oh good catch, how strange. IDEs gone rogue I think!

Reply:
Apropos - what *is* the format we're using here called? Then I can probably configure IDEA to use that.